internal/kafka/internal/handlers/admin_kafka.go

```diff
@@ -20,14 +20,14 @@ import (
 )
 
 type adminKafkaHandler struct {
-	service        services.KafkaService
+	kafkaService   services.KafkaService
 	accountService account.AccountService
 	providerConfig *config.ProviderConfig
 }
 
-func NewAdminKafkaHandler(service services.KafkaService, accountService account.AccountService, providerConfig *config.ProviderConfig) *adminKafkaHandler {
+func NewAdminKafkaHandler(kafkaService services.KafkaService, accountService account.AccountService, providerConfig *config.ProviderConfig) *adminKafkaHandler {
```

Comment:
This is not related to this PR, but why an exported function (NewAdminKafkaHandler) is returning a non exported type (adminKafkaHandler)?

Reply:
not sure. I can change it if you think it should be exported

Reply:
I don't think it makes sense that an exported function returns a non exported value: how can code calling this function from outside the package deal with the return value?

I believe visibility should be coherent between functions and return values.

@machi1990 @pawelpaszki  WDYT?

Reply:
yes makes sense to me. @machi1990 - if you agree, I will make this change

Reply:
I think @ziccardi argument makes sense. But I'll defer from doing the change in this PR and create a follow issue to address this, plus do the same for the other handlers as well if it is needed: so far, the handlers have not been really exported, and the route_loader.go (the only place where the handlers are being used) is able to interact with the handlers without any issue.